tutorials/containers/install-openstack-with-conjure-up/install-openstack-with-conjure-up.md

```diff
@@ -0,0 +1,178 @@
+---
+id: install-openstack-with-conjure-up
+summary: Learn how to deploy OpenStack on a single machine using the conjure-up spell.
+categories: containers
+tags: containers, server, openstack, conjure-up, gci
+difficulty: 2
+status: published
+feedback_url: https://github.com/canonical-websites/tutorials.ubuntu.com/issues
+published: 2018-01-13
+author: Konrad Krawiec <konrad11901@gmail.com>
+
+---
+
+# Install single-server OpenStack with conjure-up
+
+## Overview
+Duration: 1:00
+
+[OpenStack](https://www.openstack.org) is a very popular software for creating private and public clouds.
+
+OpenStack is quite an advanced platform and deploying it may not be an easy task. Luckily, with [conjure-up](https://conjure-up.io), it's very simple and straightforward.
+
+### What you'll learn
+
+- How to install OpenStack on your machine with conjure-up
+- How to install and configure LXD
+
+### What you'll need
+
+- A well-specced computer running Ubuntu 16.04 LTS with 16 gigabytes of RAM (SSD drive is strongly recommended)
+- Some basic command-line knowledge
+- An hour of your time
+
+## Installing and configuring LXD
+Duration: 3:00
+
+We'll need to set up [LXD](https://linuxcontainers.org/lxd) to deploy OpenStack.
+
+negative
+: **LXD version**
+conjure-up requires that the minimum version of LXD be **2.18**, so the version of LXD distributed through official repositories (which can be installed with `sudo apt install lxd`) is **not** compatible with conjure-up.
+
+### Installing LXD
+
+The recommended installation method of LXD are [snaps](https://snapcraft.io). Don't worry - the snap support is included with Ubuntu 16.04 LTS and above, so we don't have to do anything.
```

Comment:
I'd write something more like *The recommended LXD installation method is to use snaps...*